_episodes/l2-04-colaborative-development.md

```diff
@@ -34,37 +34,53 @@ knowledge acquired so far.
->  yourself or one of your colleagues as responsible for each of them.
->  Remember that if you do not have write access to the repository (which is the case here)
->  and you are not part of the same organisation of the repository, the only way of being
->  assigned to an Issue is by making a comment on the Issue.
-> - Fork the administrator's repository. Administrator, did you notice how the
->  number of forks increases? Which GitHub users forked it from you? And from
->  the original repo?
-> - Work on the recipes you have been assigned. Practice the concepts learnt
->  in previous episodes about cloning a repository, making the changes locally
->  and pushing those changes back to the remote repository. You can even try a
->  [gitflow approach](https://nvie.com/posts/a-successful-git-branching-model/)
->  if you feel ambitious!
-> - When ready, open a PR to the administrator's repo and request their review.
-> - Administrator, review the PR, request some changes and accept others. Make sure the
->   relevant checks performed by the continuous integration system are all passing.
+> **Create a new release** (Administrator's task)
+>
+> - Create a new release, let's say `1.0.0`, as the starting point of the book.
+>
+> **Creating new issues** (to be done by everyone):
+>
+> - Open new issues with recipes for sauces you would like to have in the book.
+>
+> **Explore existing issues** (Contributor's task):
+>
+> - Comment on the issues that you want to be assigned to. Remember, you neither have write access to the repository nor you are not part of the same organisation of the repository (in this case the repository belongs to the Administrator is not a part of an organisation), the only way of being assigned to an issue is by making a comment on the issue.
+>
+> **Task assignment** (Administrator's task):
+>
+> - Add some tags, prioritise some of the recipes, and assign yourself or one of your colleagues as responsible for each of them.
+>
+> **Working on a branch, pushing the changes, and opening a PR** (Contributor's tasks):
+>
+> - Clone the Administrator's repository.
+> - Create a branch and work on the recipes you have been assigned. Practice the concepts learnt in previous episodes about making the changes locally and pushing those changes back to the remote repository. You can even try a [gitflow approach](https://nvie.com/posts/a-successful-git-branching-model/) if you feel ambitious!
+> - When ready, open a PR to the Administrator's repo and request their review.
+>
+> **Reviewing and Merging PRs** (Administrator's tasks):
+>
+> - Review the PR, request some changes and accept others. Make sure the relevant checks performed by the continuous integration system are all passing.
 > - When ready, merge the PR.
-> - Whenever there is a new recipe added, the administrator creates a new release.
+>
+> **Creating a new release** (Administrator's task):
+>
+> - Whenever there is a new recipe added, create a new release.
```

Comment:
Do we want them to be making so many releases? They make one release at the start -- would it be enough to have them just make another release at the end?

Reply:
Probably one at the start and another one later is enough

Reply:
I can see what you mean